charts/kube-green/values.yaml

```
@@ -29,6 +29,13 @@ kubeRbacProxy:
     repository: gcr.io/kubebuilder/kube-rbac-proxy
     pullPolicy: IfNotPresent
     tag: "v0.15.0"
+  resources:
+    limits:
+      cpu: 400m
+      memory: 400Mi
+    requests:
+      cpu: 100m
+      memory: 50Mi
```

I update in this way to be the same as configured previously

```suggestion
    limits:
      cpu: 500m
      memory: 128Mi
    requests:
      cpu: 5m
      memory: 64Mi
```